app/views/home/index.html.erb

```diff
@@ -1,50 +1,24 @@
-<% if current_user %>
-  <div class='container'>
-    <h3> Logged In as: <%= current_user.email %> </h3>
-      <%= link_to 'Log Out', session_path(:user_id),
-      data: {turbo_method: :delete, turbo_confirm: "Are you sure?"}, class: 'btn  ' %>
-
-    <h2>Add a Note:</h2>
-    <%= render 'messages/form' %>
-
-    <h2>Notes</h2>
-    <%= render current_user.messages %>
+<nav class="navbar navbar-expand-sm navbar-dark bg-dark">
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarsExample03" aria-controls="navbarsExample03" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+
+  <div class="collapse navbar-collapse" id="navbarsExample03">
+    <ul class="navbar-nav mr-auto">
+      <li class="nav-item">
+        <%= link_to 'Sign Up', new_user_path, class: 'nav-link' %>
+      </li>
+      <li class="nav-item">
+        <%= link_to 'Log In', new_session_path, class: 'nav-link' %>
+      </li>
+    </ul>
   </div>
+</nav>
 
-<% else %>
-
-  <div class = "container">
-
-    <div class = "row startbg">
-      <div class = "col-6 m-auto p-3" id = "start">
-        <h3>Want to LOGIN or SIGNUP?</h3>
-      </div>
-    </div>
-
-    <div class = "row">
-      <div class = "col-6 m-auto p-2" id = "menu">
-        
-        <ul class = "signbg">
-          <li><p><%= link_to 'Sign Up', new_user_path %></p></li>
-          <li><p>or</p></li>
-          <li><p><%= link_to 'Log In', new_session_path %></p></li>
-        </ul>
-      </div>
-    </div>
-
-    <div class = "row whitbg">
-      <div class = "col-sm-6 m-auto p-4" id = "content">
-        <p>Like we said, people don’t enjoy filling out forms. Designers have to create forms
-           that are so easy to fill out that the user doesn’t have to do much work.
-           This leads us to Medium’s signup page. After clicking the “Get started” call to
-           action button on the homepage, a modal pops up. You see an engaging illustration for
-           visual interest. But what is striking is there are no input fields. Are we on to a
-           winner already?
-        </p>
-      </div>
+<div class="container">
```

Comment:
HOW WOULD YOU UNDERSTAND WE NEED NOTHING ON THIS PAGE??????

Reply:
done